src/input/styled-components.js

```diff
@@ -9,6 +9,31 @@ import {styled} from '../styles/index.js';
 import type {ThemeT} from '../styles/types.js';
 import {ADJOINED, ENHANCER_POSITION, SIZE} from './constants.js';
 import type {SharedPropsT} from './types.js';
+import {DeleteAlt} from '../icon/index.js';
+
+export const StyledClearIconContainer = styled(
```

Comment:
please type this - I understand it's not showing errors now, but it will [show errors](https://github.com/uber-web/baseui/pull/1417) in consumer code. make [this](https://github.com/uber-web/baseui/pull/1390/files#diff-a9d0201e8c5bdc946e54218bb9206174R2) change to test locally

Reply:
i _think_ i resolved the type issue here, but changing the config didn't make vscode report any problems with the un-typed version so not sure.

Reply:
I suppose we can always push a quick patch if necessary